### ISSUE TITLE
Rename repo

### ISSUE THREAD

**Issue by tammersaleh** [cloud-gov/cg-site#1593]
Since we're now in a dedicated `cloud-gov` org, we should rename this to either `site`, or `docs.cloud.gov` to match the domain it's deployed to.

**Comment by eddietejeda** [comment]
I favor https://github.com/cloud-gov/www.cloud.gov

Warning: changing the repo name will impact the federalist bucket name.

**Comment by tammersaleh** [comment]
> Warning: changing the repo name will impact the federalist bucket name.

can you elaborate?

**Comment by eddietejeda** [comment]
> can you elaborate?

The site will break and we'll need to go through security scan process with IT again.

@amirbey can you confirm?

**Comment by amirbey** [comment]
break ... its more like freeze. changing your repository name will require you to add a new site to federalist. each repository on federalist has its own bucket and then your cdn needs to point to the origin and path of where you're renamed repository will build.


yes ... changing the repo name will require you to go through the Site Review process ... which includes a scan.

your cdn should reside in the federalist org and space along with the other sites. perhaps this is an opportunity to set this up properly.

**Comment by pburkholder** [comment]
Would we also have to redo the HSTS work on a rename?

**Comment by amirbey** [comment]
Yes. We would need to update the the bucket name in the proxy for the HSTS work implemented by @apburnes

**Comment by markdboyd** [comment]
Repo is being archived, closing this issue.
